controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -1469,6 +1495,24 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           };
         }
 
+        const memberCount = await orgRepo.memberCount(authContext.organizationId);
+
+        const usersFeature = await orgRepo.getFeature({
+          organizationId: authContext.organizationId,
+          featureId: 'users',
+        });
+
+        const limit = usersFeature?.limit === -1 ? undefined : usersFeature?.limit;
+
+        if (limit && memberCount >= limit) {
+          return {
+            response: {
+              code: EnumStatusCode.ERR_LIMIT_REACHED,
+              details: `The user limit for this organization has been reached`,
```

Comment:
I'd explain what to do next.

Reply:
This message is not show to the user since the invite form isn't available when limits have been reached.